torch/_inductor/fx_passes/pre_grad.py

```diff
@@ -340,3 +344,59 @@ def transpose_matmul(A: torch.Tensor, B: torch.Tensor, Atrans: bool, Btrans: boo
     if Btrans:
         B = B.transpose(-1, -2)
     return torch.matmul(A, B)
+
+
+_pointwise_ops = [
+    operator.add,
+    operator.sub,
+    operator.mul,
+    operator.iadd,
+    operator.isub,
+    operator.imul,
+    operator.itruediv,
+    operator.ifloordiv,
+]
```

Comment:
Need a better way to deal with this, looking for suggestion.

Reply:
I think it will probably be better to invest in unbacked symint compilation support in inductor instead